packages/react-native/src/migrations/update-14-5-8/change-searchDir-storybook.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { addProjectConfiguration, getProjects, Tree } from '@nrwl/devkit';
+import { createTreeWithEmptyV1Workspace } from '@nrwl/devkit/testing';
+import update from './change-searchDir-storybook';
+
+describe('change-searchDir-storybook', () => {
+  let tree: Tree;
+
+  beforeEach(async () => {
+    tree = createTreeWithEmptyV1Workspace();
+    addProjectConfiguration(tree, 'products', {
+      root: 'apps/products',
+      sourceRoot: 'apps/products/src',
+      targets: {
+        storybook: {
+          executor: '@nrwl/react-native:storybook',
+          options: {
+            searchDir: 'apps/products',
+          },
+        },
+      },
+    });
+  });
+
+  it(`should change searchDir from string to array`, async () => {
+    await update(tree);
+
+    getProjects(tree).forEach((project) => {
```

Comment:
There should just be one project here, let's use `readProjectConfiguration` function so the `expect` is not nested in the `forEach` function.

I'm wary of nested expects in case the projects is empty (which it shouldn't TBH), the expect will not execute.

Reply:
changed.